src/lib.rs

```diff
@@ -206,11 +216,9 @@ impl From<String> for Value {
     }
 }
 
-impl From<&'static str> for Value {
-    fn from(v: &'static str) -> Value {
-        let mut vec = Vec::new();
-        vec.extend_from_slice(v.as_bytes());
-        Value(vec)
+impl From<&str> for Value {
```

Comment:
If I understand it correctly, the extra static lifetime requirement is trying to avoid copying without user actually noticing it. Creating Key/Value from &str can be done by calling to_string() explicitly or simply move the String where the &str was created from.

Reply:
Right. :) The reason `&'static` is supported is because this is almost always a `const` or a raw string in the code, which the user would expect to be cloned to be used in this way.

Maybe it should be changed? 🤔

Reply:
@Kerollmops Any chance you have time to follow up on this?

Reply:
The problem is that it will not fix the problem, the `Value` type require a `Vec` internally, so the static str will need to be transformed into a `Vec` anyway.

Reply:
Right, but the intention is that `&'static str` are used for examples or testing code, we don't want people to use it for `&'a str` or other random borrows, just for, for example

This is ok:

```rust
let test_key = "blah"`;
Value::from(test_key);
```

While this shouldn't work, since it introduces invisible copying of data which is probably real:

```rust
let test_key = String::from("blah");
Value::from(&test_key)
```